packages/@aws-cdk/aws-ecr-assets/test/test.image-asset.ts

```diff
@@ -87,6 +87,23 @@ export = {
     test.done();
   },
 
+  'with file'(test: Test) {
+    // GIVEN
+    const stack = new Stack();
+
+    const directoryPath = path.join(__dirname, 'demo-image-custom-docker-file');
+    // WHEN
+    new DockerImageAsset(stack, 'Image', {
+      directory: directoryPath,
+      file: 'Dockerfile.Custom'
+    });
+
+    // THEN
+    const assetMetadata = stack.node.metadata.find(({ type }) => type === ASSET_METADATA);
+    test.deepEqual(assetMetadata && assetMetadata.data.file, path.join(directoryPath, 'Dockerfile.Custom'));
```

Comment:
This assertion should have just been:

```suggestion
    test.deepEqual(assetMetadata && assetMetadata.data.file, 'Dockerfile.Custom');
```